modyn/tests/supervisor/internal/test_grpc_handler.py

```diff
@@ -349,135 +365,67 @@ def test_wait_for_evaluation_completion(*args):
     handler = GRPCHandler(get_simple_config())
     handler.init_cluster_connection()
     assert handler.evaluator is not None
-    eval_status_queue = mp.Queue()
-    evaluations = {
-        1: EvaluationStatusReporter(
```

Comment:
Since I removed those fields `num_samples_seen`, etc. (the number to report the internal progress of evaluation on one interval), this queue is no longer used.